torchvision/models/densenet.py

```diff
@@ -139,8 +149,16 @@ class DenseNet(nn.Module):
           but slower. Default: *False*. See `"paper" <https://arxiv.org/pdf/1707.06990.pdf>`_
     """
 
-    def __init__(self, growth_rate=32, block_config=(6, 12, 24, 16),
-                 num_init_features=64, bn_size=4, drop_rate=0, num_classes=1000, memory_efficient=False):
+    def __init__(
+        self,
+        growth_rate: int = 32,
+        block_config: Tuple[int] = (6, 12, 24, 16),
```

Comment:
mypy is complaining here. `Tuple` need to be specified with the exact number of elements.

Reply:
This should either be `Tuple[int, int, int, int]` if we are sure that this will always be four elements or `Tuple[int, ...]` if the length varies.

Reply:
All standard cases are `Tuple[int, int, int, int]`, and while the code might work with others, it's not the default so I would say to leave it with a fixed number

Reply:
Fixed it!